backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/BpkAppSearchModal.kt

```diff
@@ -79,7 +79,7 @@ fun BpkAppSearchModal(
     clearAction: BpkClearAction,
     modifier: Modifier = Modifier,
     state: BpkModalState = rememberBpkModalState(),
-    behaviouralCallback: BehaviouralCallback? = null,
+    behaviouralEventWrapper: (@Composable (BpkItem, content: @Composable BpkClickHandleScope.() -> Unit) -> Unit)? = null,
```

Comment:
i don't know if there's a way to have this as a reusable declaration, rather than having to define this api again and again. but like this it has the benefit that we can actually provide the right type as well, rather than just any

Reply:
a `typealias`?

Reply:
yup :D i somehow doubted this was gonna work with composable functions, but it does